src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -375,10 +376,10 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 		}
 	}
 
-	private initTabComponents(value: IDashboardTab): { id: string; title: string; container: object; alwaysShow: boolean; } {
+	private initTabComponents(value: IDashboardTab): { id: string; title: string; container: object; alwaysShow: boolean; iconClass?: string } {
```

Comment:
>{ id: string; title: string; container: object; alwaysShow: boolean; iconClass?: string } [](start = 50, length = 89)

Would suggest declaring this as a type somewhere - makes it a lot easier to parse/reuse

Reply:
this is only used once here by this private method, i am going to leave it as is.